demisto_sdk/commands/validate/validators/WZ_validators/WZ104_is_wrong_link_in_wizard.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.handlers import DEFAULT_JSON5_HANDLER as json
+from demisto_sdk.commands.content_graph.objects.wizard import Wizard
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Wizard
+
+
+class IsWrongLinkInWizardValidator(BaseValidator[ContentTypes]):
+    error_code = "WZ104"
+    description = "Checks if all fetch integrations have playbook."
+    rationale = "Ensuring all integrations linked to playbook."
+    error_message = "The Following integrations has not linked playbook: {0}"
+    related_field = "fetching_integrations"
+    is_auto_fixable = False
+
+    def integrations_without_playbook(self, content_item: ContentTypes) -> List[str]:
+        content_item_json = json.loads(content_item.text)
+        wizard_json_object = content_item_json.get("wizard", {})
+        set_playbooks = wizard_json_object.get("set_playbook", [])
+        playbooks_link_to_integration = {
```

Comment:
naming tip: currently this variable sounds like a mapping from `playbook links` to `integrations`
if you want to specify the mapping in the dict name, make sure it's `A_to_B`, e.g. `playbook_to_integration_link` (or even drop the `_link` suffix for a more simple name)